install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-LUA_MOD_DIR="./lua-mod"
-LIB_PATH="/usr/local/crowdsec/lua/haproxy/"
+LUA_MOD_DIR="."
```

Comment:
Could you please just double check with @sabban about where those files should be to understand who of code or documentation is wrong (if not both :D )